Packs/FeedAzureADConnectHealth/Integrations/AzureADConnectHealthFeed/README.md

```diff
@@ -0,0 +1,10 @@
+## Microsoft Azure AD Connect Health web scraper:
```

Comment:
in the readme we put the generation of the sdk command: `generate-docs`. please see:
https://xsoar.pan.dev/docs/integrations/integration-docs

Reply:
Done